src/components/ScreenWrapper.js

```diff
@@ -126,6 +131,7 @@ class ScreenWrapper extends React.Component {
                                     : this.props.children
                             }
                             <KeyboardShortcutsModal />
+                            {this.props.isSmallScreenWidth && <OfflineIndicator />}
```

Comment:
Looks like this is not respecting the safe area padding which is why the indicator appears cut off on mobile and too far down on web. I'm not sure which approach to take exactly but the safe area padding needs to be applied when the OfflineIndicator is present for this to look right

Reply:
@arosiclair Nice catch! 

Weird tho, because for both web and mobile, safe area padding should already be applied from [this](https://github.com/Expensify/App/blob/79836db43a93c449fa62f4728d1e58c815b89993/src/components/ScreenWrapper.js#L100-L117). I'll take a look.

Reply:
^^ Not actually because of safe area padding, see my comment below.